crates/fuel-core/src/database/storage.rs

```diff
@@ -0,0 +1,96 @@
+use crate::database::{
+    Column,
+    Database,
+};
+use fuel_core_storage::{
+    tables::FuelBlockRoots,
+    Error as StorageError,
+    Mappable,
+    Result as StorageResult,
+    StorageInspect,
+    StorageMutate,
+};
+use fuel_core_types::blockchain::primitives::BlockHeight;
+use serde::{
+    de::DeserializeOwned,
+    Serialize,
+};
+use std::borrow::Cow;
+
+/// The trait shows that the database table has a corresponding Merkle root table.
+trait HasMerkleRoots: Mappable {
```

Comment:
What are these private traits for? I don't understand how they are related to the block importer? Was this left here back accident?

Reply:
Yep, I forgot to remove it, thank you